### PR TITLE
log filename of unparsable mysql schema file

### LINKED ISSUE #245: mysql: generate command error doesn't include enough info to help debug
Running `sqlc` in mysql mode gets me the error:

```
$  sqlc generate
# package sqlcmodels
error parsing file: schema failure: syntax error at position 1362
```

But it's not clear what file wasn't parsed correctly.

### PR DESCRIPTION
**Pull request by jmhodges**
Updates #245

### CODE REVIEW

File: internal/mysql/parse.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ func parsePath(sqlPath string, inPkg string, s *Schema, settings dinosql.Generat
 		}
 		queries, err := parseContents(filename, string(contents), s, settings)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("Failed to parse contents of file [%v]: %s", filename, err)
```
Comment (kevinburkemeter) [inline internal/mysql/parse.go]: Think you want to swap `%v` and `%s` here. (Also maybe use the new `%w` thing?)

Comment (jmhodges) [reply internal/mysql/parse.go]: That's up to @kyleconroy! I was using the same pattern of arguments from the errors right above it.

Comment (kevinburkemeter) [reply internal/mysql/parse.go]: Ah fair!

Comment (kyleconroy) [reply internal/mysql/parse.go]: Both of you are correct! sqlc requires Go 1.13, so we should be using `%w` for all wrapped error.

**Comment by kyleconroy** [comment]
Thanks @jmhodges! Once you update` %s` to `%w I'll merge this in.

File: internal/mysql/parse.go (hunk 1 above)
Comment (jmhodges) [reply internal/mysql/parse.go]: beep boop all done!

**Comment by jmhodges** [comment]
For reasons, I also fixed the other places that could use %w in https://github.com/kyleconroy/sqlc/pull/250

**Pull request merged**